api/lib/mno_enterprise/concerns/controllers/jpi/v1/admin/base_resource_controller.rb

```diff
@@ -33,14 +29,14 @@ def check_authorization
   end
 
   def render_not_found(resource = controller_name.singularize, id = params[:id])
-    render json: { errors: {message: "#{resource.titleize} not found (id=#{id})", code: 404, params: params} }, status: :not_found
+    render json: { errors: { message: "#{resource.titleize} not found (id=#{id})", code: 404, params: params } }, status: :not_found
   end
 
   def render_bad_request(attempted_action, issue)
-    render json: { errors: {message: "Error while trying to #{attempted_action}: #{issue}", code: 400, params: params} }, status: :bad_request
+    render json: { errors: { message: "Error while trying to #{attempted_action}: #{issue}", code: 400, params: params } }, status: :bad_request
   end
 
   def render_forbidden_request(attempted_action)
-    render json: { errors: {message: "Error while trying to #{attempted_action}: you do not have permission", code: 403} }, status: :forbidden
+    render json: { errors: { message: "Error while trying to #{attempted_action}: you do not have permission", code: 403 } }, status: :forbidden
```

Comment:
Can't you put this 3 methods in application_controller.rb, so that you dont' have to repeat them as well in the non_admin base controller?

Reply:
Done.